src/Library/Ui/Premium.php

```diff
@@ -0,0 +1,55 @@
+<?php //phpcs:ignore WordPress.Files.FileName
```

Comment:
I don't understand this file. Is it doing the same thing as https://github.com/BoldGrid/boldgrid-backup/blob/master/admin/class-boldgrid-backup-admin-dashboard.php ?